initfiles/sbin/hpcc-run.sh.in

```diff
@@ -52,134 +52,144 @@ getIPS(){
 }
 
 getDali(){
-        DIP=`${INSTALL_DIR}/sbin/configgen -env ${envfile} -listall | grep Dali | awk -F, '{print \$3}'  | sort | uniq`
+    DIP=`${INSTALL_DIR}/sbin/configgen -env ${envfile} -listall | grep Dali | awk -F, '{print \$3}'  | sort | uniq`
 }
 
 createIPListFile(){
-    _file=$1
-    echo "$IPS" > $_file
+    local _output=$1
+    echo "$IPS" > $_output
 }
 
 createIPListFileExcludeDIP(){
-  _file=$1
-  echo "$IPS" | grep -v $DIP  > $_file
+    local _input=$1
+    local _output=$2
+    rm -f $_output >/dev/null 2>&1
+    touch $_output
```

Comment:
originally I was using this because on line 230 we use wc -l to check to insure that there was actually contents in the ip file that excluded the DIP.  I think I might have a better way to write this though.